_includes/resource-table-all.html

```diff
@@ -142,7 +142,7 @@ <h3 class="{% if site.theme_variables.headings.resource-table-all-collapse %}mb-
         <thead>
             <tr class="text-nowrap">
                 <th>Tool or resource
-                    <a data-bs-toggle="tooltip" data-bs-original-title="This is a curated list which means that not all tools or resources that exist for this topic are listed here. This is mainly because we do not intend to be a registry. In most cases you will only find back the tools or resources that are mentioned in this page.">
+                    <a data-bs-toggle="tooltip" data-bs-original-title="This is a curated subset of (and not a comprehensive list of all) tools and resources on this topic - you'll typically find only the tools or resources referenced on this page.">
```

Comment:
Good catch! Actually this is the tooltip foor the national tools, I don't know if it is relevant here :thinking: maybe we should move this tool the general tools above!

Reply:
Yes, I was wondering about that too. It is definitely not just related to the national resources.

Reply:
I have also noticed a few other small things around "Additional information" and "national resources" sections - I'd be happy to have a quick chat when you are free to show you want I mean. Otherwise, I could just raise issues and add screenshots - but I think a chat would be better if you like.

Reply:
I will contact you on slack!

Reply:
Just realize that I don't find you, matrix?